INSTALLING.md

```diff
@@ -70,19 +70,31 @@ docker-compose up -d
 
 PostgreSQL and Redis will now set itself up in the background, and will be available in
 a minute or so.
-You'll need to run the `docker-compose up` command again if you
-reboot your computer to start the database again.
-
+You'll need to run the `docker-compose up` command again (in a Nix-shell) if you
+reboot your computer to start the database again. If you like, you can run the
+container in the background by adding `-d` to the mentioned command.
 If you're already running a copy of PostgreSQL, you can use this too.
-You'll need to create a user with all privileges for the
-databases `koala-development` and `koala-test`, and set this in the `.env` file.
 
-There is an example file in the root of this repository called
+The previous command starts a process which should keep running while running Koala.
+In addition, the following dependency should also be kept running:
```

Comment:
This should not strictly necessary when not working on the front-end.
I can imagine most people want hot reloading though, so maybe we should leave this as-is.

Reply:
I also think we should try to set up a Nix environment with `foreman` or similar, which would give a oneline command to start the databases (Docker), the dev-server and Koala itself.
That would improve the developer first-time experience quite a bit, I'll look into this later.

Reply:
> This should not strictly necessary when not working on the front-end. I can imagine most people want hot reloading though, ...

I will mention what you said here, so that people can choose for themselves.